internal/scene/scene.go

```diff
@@ -55,23 +55,6 @@ func SetWindowForAllScenes(win *pixelgl.Window) {
 	window = win
 }
 
-// TODO: This has probably been copied here as a reference.
```

Comment:
Yeah, housekeeping :+1:

Reply:
Yup, I take it it's fine now, with the commented code removed, if I'm not mistaken, so resolving this conversation, too. :)